### PR TITLE
i245--VCPKG_instructions: 

### LINKED ISSUE #245: Add exact VCPKG install instructions
**Is your feature request related to a problem? Please describe.**
We tells users to install the dependencies to GB via VCPKG, but we don't provide the exact commands to install.
This can make things difficult for people new to VCPKG

**Describe the solution you'd like**
Provide exact VCPKG + visual studio instructions

**Describe alternatives you've considered**
only provide instructions for VCPKG and don't worry about Visual Studio instructions

### PR DESCRIPTION
**Pull request by lavinrp**
Closes #245 

update docs with better VCPKG instructions

### CODE REVIEW

File: Guides/DeveloperGuide.md [hunk 1]
```diff
@@ -2,24 +2,25 @@
...
-      - [2.2.2: CMake Settings](#222-cmake-settings)
-      - [2.2.3: Set Working Directory](#223-set-working-directory)
-      - [2.2.4: Set Environment (Linux Only)](#224-set-environment-linux-only)
-  - [3: Running Tests](#3-running-tests)
+  - [2: Dependencies](#2-dependencies)
+  - [3: IDE Setup](#3-ide-setup)
     - [3.1: Visual Studio](#31-visual-studio)
+      - [3.1.1: Download C++](#311-download-c)
+      - [3.1.2: Download Extensions (optional)](#312-download-extensions-optional)
+      - [3.1.3: CMake Settings](#313-cmake-settings)
```
Comment (gyurgyma) [inline Guides/DeveloperGuide.md:10]: Should we also add the run settings? In particular, I've had to do it for non-vcpkg libs.

File: Guides/InstallGuide.md [hunk 2]
```diff
@@ -27,17 +29,24 @@ This guide will show you how to
...
+   * [Windows](https://github.com/AgreeableDeerGames/ag_vcpkg#quick-start-windows)
+   * [Unix](https://github.com/AgreeableDeerGames/ag_vcpkg#quick-start-windows)
+2. install sfml
+   * `vcpkg install sfml:[vcpkg-triplet-here]`
+3. install tgui (required for demo only)
+   * `vcpkg install tgui:[vcpkg-triplet-here]`
+4. install box2d (required for demo only)
+   * `vcpkg install box2d:[vcpkg-triplet-here]`
+5. install boost (required for tests only)
+   * `vcpkg install boost:[vcpkg-triplet-here]`
```
Comment (gyurgyma) [inline Guides/InstallGuide.md:42]: We should discuss what triplet we use yes? I think so.

Comment (gyurgyma) [reply Guides/InstallGuide.md:42]: Perhaps beforehand or afterwards.

File: Guides/InstallGuide.md [hunk 3]
```diff
@@ -27,17 +29,24 @@ This guide will show you how to
...
 
 #### 2.2.2: custom install
 Build and install the following dependencies on your own. 
 * [SFML](https://www.sfml-dev.org/download/sfml/2.5.1/)
-* [Box2D](https://box2d.org/) (optional for demo)
-* [TGUI](https://tgui.eu/) (optional for demo)
-* [Boost](https://www.boost.org/users/download/) (optional for tests)
+* [Box2D](https://box2d.org/) (required for demo only)
```
Comment (gyurgyma) [inline Guides/InstallGuide.md:47]: I think this is optional for demo even...

**Comment by lavinrp** [comment]
@Christopher-Ochs @davidgreiner 
Adding you two as reviewers since you have gone through the setup process most recently.

Your feedback on this would be greatly appreciated.

File: Guides/DeveloperGuide.md (hunk 1 above)
Comment (lavinrp) [reply Guides/DeveloperGuide.md:10]: What do you mean by run settings?

File: Guides/InstallGuide.md (hunk 2 above)
Comment (lavinrp) [reply Guides/InstallGuide.md:42]: Agreed.
I'll add that.

File: Guides/InstallGuide.md (hunk 3 above)
Comment (lavinrp) [reply Guides/InstallGuide.md]: Now mention that this is only required for the platform demo.
How do you like the wording?

File: Guides/InstallGuide.md (hunk 2 above)
Comment (gyurgyma) [reply Guides/InstallGuide.md]: Fine by me.

File: Guides/DeveloperGuide.md (hunk 1 above)
Comment (gyurgyma) [reply Guides/DeveloperGuide.md:10]: By "run settings", I mean the options we use for "Run-no debug" and the other launch settings in launch.vs.json. We can always add them later if we like.

File: Guides/InstallGuide.md (hunk 3 above)
Comment (gyurgyma) [reply Guides/InstallGuide.md]: That's fine.

**Review by gyurgyma — APPROVED** [review]

**Review by Christopher-Ochs — APPROVED** [review]

File: Guides/DeveloperGuide.md (hunk 1 above)
Comment (lavinrp) [reply Guides/DeveloperGuide.md:10]: lets add those in a future enhancement.

**Pull request merged**